include/myst/kernel.h

```diff
@@ -226,7 +226,7 @@ void myst_start_shell(const char* msg);
 
 const char* myst_syscall_str(long n);
 
-MYST_INLINE bool myst_is_bad_addr(const void* ptr)
+MYST_INLINE bool myst_is_good_addr(const void* ptr)
```

Comment:
I have similar fix in #724. Suggest a better naming
`myst_is_within_kernel`
or
`myst_is_within_enclave` (to avoid the confusion that user memory is also within this range)

Reply:
This check is more restrictive than just "within-enclave" so I like your first suggestion: `myst_is_within_kernel`.

Reply:
Also, the kernel should not know about the concept of an enclave. :)

Reply:
Added this, thank you!